crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -166,6 +166,12 @@ pub struct CheckpointSummary {
     pub sequence_number: CheckpointSequenceNumber,
     pub content_digest: CheckpointContentsDigest,
     pub previous_digest: Option<CheckpointDigest>,
+    /// If this checkpoint is the last checkpoint of the epoch, we also include the committee
+    /// of the next epoch. This allows anyone receiving this checkpoint know that the epoch
+    /// will change after this checkpoint, as well as what the new committee is.
+    /// The committee is stored as a vector of validator pub key and stake pairs. The vector
+    /// should be sorted based on the Committee data structure.
```

Comment:
why is it required to be sorted? is it safer to use a structure that guarantees the order?